SimpleWasm/wwwroot/appsettings.json.md

```diff
@@ -11,7 +11,7 @@ In order for you to run the sample app, you need to create an `appsettings.json`
       "discoverydocumenturi": "{URI}",
       "clientid": "{string}",
       "scope": "{string}",
-      "enablesilentauthorizationcodeflow": {true|false}
+      "enablesilentauthorizationcodeflow": true

```

Comment:
Here too. Instead of true, follow the same notation as in the others and use {bool}